CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#48](https://github.com/rubocop-hq/rubocop-minitest/pull/48): Enable to work `Exclude` in AllCops section. ([@koic][])
```

Comment:
I think the changelog entry is unnecessary.
If my understanding is correct, this change does not affect users because the configuration file does not have `Exclude`.

Reply:
Sure!